src/main/java/core/basesyntax/Application.java

```diff
@@ -2,6 +2,9 @@
 
 public class Application {
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+        int numberOfBalls = Lottery.getBallsAmount();
+        for (int i = 0; i < numberOfBalls; i++) {
+            System.out.println(new Lottery().getRandomBall());
```

Comment:
Don't create a `Lottery()` inside the loop, instead declare and init it before the loop. See [here](https://mate-academy.github.io/jv-program-common-mistakes/java-core/oop/lottery#dont-create-several-instances-of-the-same-class-if-you-can-use-only-one-instance-for-your-purposes).